Dev-Corner/slides/tool_integration.html

```diff
@@ -968,6 +964,68 @@
 
                     <section data-markdown>
                         <script type="text/template">
+                            <img src="../../shared/images/conda_logo.png" class="no-border"  height=25 />
+                            ### Channels
+
+                            Packages are provided through Channels within Continuum.Anaconda. It's the way to contribute.
```

Comment:
Continuum own Anaconda.
I wanted to introduce this term continuum because it appear in the different url contacted by Conda.